proposals/lip-0054.md

```diff
@@ -161,7 +180,8 @@ stateRecoveryParams = {
         "siblingHashes": {
             "type": "array",
             "items": {
-                "dataType": "bytes"
+                "dataType": "bytes",
+                "length": HASH_LENGTH
```

Comment:
Just a small inconsistency but here you use 'length' but in the other params you are using 'minLength' and 'maxLength'.